packages/docusaurus/src/webpack/client.ts

```diff
@@ -4,6 +4,8 @@
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
  */
+
+import chalk from 'chalk';
```

Comment:
Although it is recommended that we use the CommonJS module format (`import chalk = require('chalk');`) to import chalk, but when using this style, tests will fail with such an error:

```
 `import =` is not supported by @babel/plugin-transform-typescript
Please consider using `import <moduleName> from '<moduleName>';` alongside Typescript's --allowSyntheticDefaultImports option.
```
It is OK?

Reply:
Seems like this is a known limitation. That is fine then :)